### ISSUE TITLE
Week 3 assignment - trouble setting the kernel

### ISSUE THREAD

**Issue by gklarenberg** [NEONScience/DI-NEON-participants#147]
I don't know if this is the correct place to give a tip. Feel free to move this if not...
I noticed that some notebooks submitted still showed version 3.6.4. One of the things that maybe is not clear from the instructions (since it also took me a while to figure out), is that you **have to** run `source activate p35` in the Terminal/command line **before** running `python -m ipykernel install --user --name p35 --display-name "Python 3.5 NEON-RSDI"`.
This will then make sure that the name "Python 3.5 NEON-RSDI" in the dropdown menu when creating a new notebook uses 3.5.5.

**Comment by cklunch** [comment]
Yes, a few people have been having this problem, thanks for sharing a solution!

**Comment by cklunch** [comment]
I'll see if I can update the instructions.

**Comment by gklarenberg** [comment]
I'm further along in the exercise now, and I see it does come up in "Check Python version" in the plotting exercise.

**Comment by cklunch** [comment]
Ah, right. But since it seems to be causing trouble for multiple people, I'll try to clarify it in the setup instructions.
